scripts/send-sms.js

```diff
@@ -27,6 +27,8 @@ require('../lib/senders')(config, log)
     let message = error.message
     if (error.reason && error.reasonCode) {
       message = `${message}: ${error.reasonCode} ${error.reason}`
+    } else if (error.stack) {
+      message = error.stack
```

Comment:
This helped me debug a problem while I was making the changes, thought it earned the right to stick around.